src/main/scala/ru/butik/butifactory/PushService.scala

```diff
@@ -34,18 +34,18 @@ object PushService {
   }
 }
 
-class PushService(apiKey: String) {
+class PushService(pushHost: String, pushPort: String, pushMethod: String, pushKey: String) {
   import com.twitter.conversions.time._
   private val twitter = Http.client
     .withRetryBackoff(Backoff.exponentialJittered(2.seconds, 60.seconds).take(10))
-    .withTransport.tls("fcm.googleapis.com")
-    .newService("fcm.googleapis.com:443")
+    .withTransport.tls(pushHost)
+    .newService(s"$pushHost:$pushPort")
 
   def pushDevice(deviceId: String, artifactVersion: ArtifactVersionAndroid): Future[Either[String, GoogleMessageResponse]] = {
-    val request = http.Request(http.Method.Post, "/fcm/send")
-    request.host = "fcm.googleapis.com"
+    val request = http.Request(http.Method.Post, pushMethod)
```

Comment:
POST is a method, but pushMethod is a path

Reply:
fixed